INSTALL

```diff
@@ -1,288 +1,368 @@
+not '/usr/local'.  It is recommended to use the following options:
+
+     ./configure --prefix=/boot/common
+
+Specifying the System Type
+==========================
+
+   There may be some features 'configure' cannot figure out
+automatically, but needs to determine by the type of machine the package
+will run on.  Usually, assuming the package is built to be run on the
+_same_ architectures, 'configure' can figure that out, but if it prints
+a message saying it cannot guess the machine type, give it the
+'--build=TYPE' option.  TYPE can either be a short name for the system
+type, such as 'sun4', or a canonical name which has the form:
+
+     CPU-COMPANY-SYSTEM
+
+where SYSTEM can have one of these forms:
+
+     OS
+     KERNEL-OS
+
+   See the file 'config.sub' for the possible values of each field.  If
+'config.sub' isn't included in this package, then this package doesn't
+need to know the machine type.
+
+   If you are _building_ compiler tools for cross-compiling, you should
+use the option '--target=TYPE' to select the type of system they will
+produce code for.
+
+   If you want to _use_ a cross compiler, that generates code for a
+platform different from the build platform, you should specify the
+"host" platform (i.e., that on which the generated programs will
+eventually be run) with '--host=TYPE'.
+
+Sharing Defaults
+================
 
-Information on the location of packages and their versions follows immediately.
+   If you want to set default values for 'configure' scripts to share,
+you can create a site shell script called 'config.site' that gives
+default values for variables like 'CC', 'cache_file', and 'prefix'.
+'configure' looks for 'PREFIX/share/config.site' if it exists, then
+'PREFIX/etc/config.site' if it exists.  Or, you can set the
+'CONFIG_SITE' environment variable to the location of the site script.
+A warning: not all 'configure' scripts look for a site script.
 
-****************** ***
-****   PACKAGES   ****
-****************** ***
+Defining Variables
+==================
 
-For rational polyhedra:
+   Variables not defined in a site shell script can be set in the
+environment passed to 'configure'.  However, some packages may run
+configure again during the build, and the customized values of these
+variables may be lost.  In order to avoid this problem, you should set
+them in the 'configure' command line, using 'VAR=value'.  For example:
 
-CoCoALib, http://cocoa.dima.unige.it/cocoalib/, version 0.99601
-MPFR,  http://www.mpfr.org/mpfr, version 4.0.1
-Flint, http://www.flintlib.org/flint, version 2.5.2
-nauty, http://users.cecs.anu.edu.au/~bdm/nauty, version 27rc2
+     ./configure CC=/usr/local2/bin/gcc
 
-For algebraic polyhedra additionally:
+causes the specified 'gcc' to be used as the C compiler (unless it is
+overridden in the site shell script).
 
-MPFR and Flint as above
-arb, https://github.com/fredrik-johansson/arb/archive/, version 2.16.0
```

Comment:
Is arb used directly or only indirectly through e-antic? @w-bruns

Reply:
It is. But perhaps it makes sense to list it here. I am not sure.